### PR TITLE
Ui tests

### PR DESCRIPTION
**Pull request by filipmihal**
I created simple ui test for testing the categories

### CODE REVIEW

File: ui_test.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        title = self.driver.find_element_by_xpath('/html/body/div/div[1]/div/div/div/h1')
+        assert title is not None
+        assert len(title.get_attribute("innerText")) > 3
+
+        #testing the categories
+        categories = self.driver.find_elements_by_class_name('card-nav-tabs')
+        assert len(categories) >= 9
+
+    def test_category_paginator(self):
+        category = Category.query.order_by(func.random()).first()
```
Comment (ochronus) [inline ui_test.py]: It looks like you're reimplementing the whole pagination logic here - one version is already in the view function. It'd make sense to factor this logic out to some kind of 'utils' or 'helpers' python file and reuse the code.

File: ui_test.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        pages = math.ceil(total_jokes / per_page)
+        remainder = total_jokes % per_page
+        if remainder == 0:
+            remainder = 10
+
+        with app.app_context():
+            category_url = url_for('categories.category', name=category.name, _external=False)
+
+        current_page_num = str(randint(1,pages))
+        url = "http://localhost:5000" + category_url + "?page=" + current_page_num
```
Comment (ochronus) [inline ui_test.py]: Instead of hardcoding the local port set it in a config variable

File: ui_test.py (hunk 1 above)
Comment (ochronus) [reply ui_test.py]: Also you could write a real unit test to test that helper function!

**Comment by filipmihal** [comment]
test

**Review by ochronus — CHANGES_REQUESTED** [review]

**Review by ochronus — COMMENTED** [review]
👍

File: config.py [hunk 3]
```diff
@@ -7,3 +7,6 @@
 SQLALCHEMY_DATABASE_URI = 'sqlite:///' + os.path.join(basedir, 'app.db')
 SQLALCHEMY_MIGRATE_REPO = os.path.join(basedir, 'db_repository')
 SQLALCHEMY_TRACK_MODIFICATIONS = False
+
+per_page = 10
+pagination_framework = 'foundation'
```
Comment (ochronus) [inline config.py]: Minor comments, feel free to ignore: config variables are in capitals by convention
Also another convention: have a newline at the end of python files

**Comment by ochronus** [comment]
Any plans on closing this PR?

**Pull request merged**